Model/Configuration.php

```diff
@@ -4,11 +4,13 @@
 
 class Configuration
 {
+		const ACCOUNT_PARAM_PATH = 'dripconnect_general/api_settings/account_param';
     const WIS_URL_PATH = 'dripconnect_general/api_settings/wis_url';
     const INTEGRATION_TOKEN = 'dripconnect_general/api_settings/integration_token';
     const MODULE_ENABLED_PATH = 'dripconnect_general/module_settings/is_enabled';
     const SALT_PATH = 'dripconnect_general/module_settings/salt';
     const LOG_SETTINGS_PATH = 'dripconnect_general/log_settings';
+		const API_TIMEOUT_PATH = 'dripconnect_general/api_settings/timeout';
```

Comment:
Do we need a dynamically set timeout? Seems like something we could set as a constant and not as a setting.

Reply:
`setTimeout` is still getting called somewhere, so I put this back so that things would load and I could run tests. I agree we don't need it anymore.